### PR TITLE
[Scene] mesh rendering

### PR DESCRIPTION
**Pull request by joeccleung**
Support Unity Materials

### CODE REVIEW

File: Final Project/Assets/Geometry/RTGeometryType.cs [hunk 1]
```diff
@@ -2,5 +2,6 @@
 {
     Nothing = 0,
     Sphere = 1,
-    Triangle = 2
```
Comment (Chaeng) [inline Final Project/Assets/Geometry/RTGeometryType.cs:5]: How necessary to have GeometryType this way?
I am using the old version of defining object type.
Sphere = 0
Triangle = 1
Do you have an alternative option for this?

**Review by Chaeng — APPROVED** [review]
Good!

File: Final Project/Assets/Geometry/RTGeometryType.cs (hunk 1 above)
Comment (joeccleung) [reply Final Project/Assets/Geometry/RTGeometryType.cs:5]: Where do you use it? This RTGeometryType lives in SRP side.

Comment (Chaeng) [reply Final Project/Assets/Geometry/RTGeometryType.cs:5]: you are right. This is .cs.

**Review by Chaeng — APPROVED** [review]

**Pull request merged**